.spellcheck.yml

```diff
@@ -0,0 +1,24 @@
+matrix:
+- name: asciidoc source py spell check configuration
+  aspell:
+    lang: en
+    d: en_US
+  sources:
+  - documentation/*.asciidoc 
+  dictionary:
+    wordlists:
+    - en-devonfw.dict
+    output: solicitor.dic
```

Comment:
why is it called `solicitor.dic`? Is that a C&P?

Reply:
yes ,, it is a C&P issue